docs/articles/nunit/release-notes/Nunit4.0-MigrationGuide.md

```diff
@@ -0,0 +1,169 @@
+---
+uid: nunit40migrationguide
+---
+
+# NUnit 4.0 Migration Guide
```

Comment:
@manfred-brands @OsirisTerje how do we feel about possibly creating a "Migration Guide" in general and counting 3.x -> 4.x as a topic within it? I'm thinking this would then be appended or preprended to for future migration guidance that will inevitably occur. If you agree I'll make the changes.

Reply:
As said, I don't mind.